service_test.go

```diff
@@ -16,9 +16,6 @@ func TestNewService(t *testing.T) {
 	svc, ok := s.(*service)
 	require.True(t, ok)
 	require.NotNil(t, svc.logger)
-	require.NotNil(t, svc.helmRenderFn)
-	require.NotNil(t, svc.yttRenderFn)
-	require.NotNil(t, svc.kustomizeRenderFn)
```

Comment:
Nit: Would be good to assert that the replacement for these three (`renderFn`) gets a default value when `NewService()` is invoked.

Reply:
done